react/Modal/index.jsx

```diff
@@ -26,12 +27,14 @@ const ModalTitle = ({ children, className }) =>
   )
 
 const ModalCross = ({ onClick, className }) => (
-  <button
-    className={classNames(styles['c-btn'], styles['c-btn--close'], styles['coz-btn-modal-close'], className)}
+  <Button
+    theme="secondary"
```

Comment:
theme='secondary' for the cross ? Should'nt it be transparent or something ? Why is is a Button instead of a button ? Why is this in this PR ?

Reply:
Oh right, should be `alpha`, not `secondary`.

> Why is is a Button instead of a button ?

Why not? I thought it would better to use our own components, don't you think?

>Why is this in this PR ?

It should have been a really quick fix, but I guess I can extract this into another PR.

Reply:
OK for me if theme is alpha. alpha IMHO is a lot less clear than transparent. Since you can have alpha=1 :/